pkg/controller/storagecluster/reconcile.go

```diff
@@ -232,6 +235,7 @@ func (r *ReconcileStorageCluster) Reconcile(request reconcile.Request) (reconcil
 		// for external cluster, we have a different set of ensure functions
 		ensureFs = []ensureFunc{
 			r.ensureCephCluster,
+			r.ensureExternalClusterConfig,
```

Comment:
@jarrpa this is the function call that we wanted to call as we discussed right?. Thanks

Reply:
Yes, but move it above `r.ensureCephCluster`.